packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -536,3 +540,197 @@ const MultipleSelectButton = <T,>({
     </>
   );
 };
+
+// ----------------------------------------------------------------------------
+
+const NativeSelect = React.forwardRef((props, forwardedRef) => {
+  const {
+    triggerProps,
+    options,
+    disabled,
+    placeholder,
+    defaultValue: defaultValueProp = placeholder !== undefined ? '' : undefined,
+    value: valueProp,
+    onChange: onChangeProp,
+    size,
+    status,
+    required,
+    ...rest
+  } = props;
+
+  return (
+    <InputWithIcon {...rest} ref={forwardedRef}>
+      <SelectButton
+        as='select'
+        size={size}
+        status={status}
+        disabled={disabled}
+        defaultValue={defaultValueProp}
+        value={valueProp === null ? '' : valueProp}
+        {...triggerProps}
+        onKeyDown={mergeEventHandlers(triggerProps?.onKeyDown, (event) => {
+          // Firefox does not open the menu on Enter, so we need to do it manually.
+          if (event.key === 'Enter') {
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
```

Comment:
Are these needed? When I removed them, I didn't get any error.

Reply:
Maybe restart your TS server? I can't get rid of this error.

![image](https://github.com/iTwin/iTwinUI/assets/9084735/396e15be-e2cd-4bc2-a4e6-b1ee0f92fdd7)

![image](https://github.com/iTwin/iTwinUI/assets/9084735/f6179f72-e4dc-45c0-ac42-73fa7bc25e9c)

Reply:
For me, it says `showPicker` exists on `currentTarget`:

<img width="472" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/53ae7b98-a288-4fca-9004-3d1ea618b546">

Reply:
I tried removing it ([`c80fd95`](https://github.com/iTwin/iTwinUI/pull/1879/commits/c80fd952316eff36c1a66f99b53da3c63c86e8b1)) and [CI started failing](https://github.com/iTwin/iTwinUI/actions/runs/8097940476/job/22130189674?pr=1879). Your IDE might be using a newer version of TS maybe.